views/cypress/tests/item-authoring-set-response.spec.js

```diff
@@ -169,5 +170,44 @@ describe('Item Authoring', () => {
             cy.get('[data-testid="save-the-item"]').click();
             cy.wait('@saveItem').its('response.body').its('success').should('eq', true);
         });
+        it('can add single interaction to an item', function () {
+            cy.get('[data-testid="manage-items"]').click();
+            cy.getSettled('[id="item-new"]').click();
+
+            cy.get('[id="item-authoring"]').click();
+            addInteraction("choice");
+        });
+        it('can add  match correct response processing to item', function () {
+            addResponseProcessing(
+                urls.itemPreview,
+                selectors.previewItemButton,
```

Comment:
no need to pass `selectors`, you can import them in `views/cypress/utils/set-response.js` and use
Will be more clear list of params